cli/pcluster/config/update_policy.py

```diff
@@ -201,3 +201,18 @@ def _check_min_count(change, patch):
     )
     + ". If you need this change, please consider creating a new cluster instead of updating the existing one.",
 )
+
+# Update extra_json
+UpdatePolicy.EXTRA_JSON = UpdatePolicy(
+    level=1000,
+    fail_reason="Updating extra_json will only affect on computefleet and not affect on head node",
+    action_needed=lambda change, patch: (
```

Comment:
Also I'd move the part on forcing the update to the fail_reason. At that point the action needed becomes the same as UNSUPPORTED which means you could put that code in common by creating a function

Reply:
Done